wpengine/faustjs
# Merl 721 faust sitemaps

## Tasks


## Description

Added back the sitemap functionality from old Faust. The config object has been modified a bit to only include the necessary options:
https://hfo8j2b0bnggvr0uthkxzg5vk.js.wpenginepowered.com/docs/reference/getSitemapProps#config

Additionally, two new docs have been added:
* [`getSitemapProps` reference doc](https://hfo8j2b0bnggvr0uthkxzg5vk.js.wpenginepowered.com/docs/reference/getSitemapProps)
* [Sitemaps Guide](https://hfo8j2b0bnggvr0uthkxzg5vk.js.wpenginepowered.com/docs/guides/sitemaps)




## Related Issue(s):



## Testing

1. Checkout the PR
2. Create `pages/sitemap.xml.js`:
   ```js
    import { getSitemapProps } from '@faustwp/core';
    
    export default function Sitemap() {}
    
    export function getServerSideProps(context) {
      return getSitemapProps(context, {
        frontendUrl: 'http://localhost:3000/',
      });
    }
   ```
3. Visit http://localhost:3000/sitemap.xml and see the sitemaps from the WP site



## Screenshots



## Documentation Changes



## Dependant PRs
